sympy/functions/combinatorial/numbers.py

```diff
@@ -1858,12 +1858,37 @@ def stirling(n, k, d=None, kind=2, signed=False):
 
 
 @cacheit
-def _nT(n, k):
+def _nT(*args):
```

Comment:
It would be possible to omit some repeated statements like `n = args[0]` if the signature were `_nT(n, k=None)`. That would take care of both 1 and 2 argument cases.